### ISSUE TITLE
Answers to questions in CSS #2

### ISSUE THREAD

**Issue by martoio** [bootsfuehrerschein-kurse/prj-rev-bwfs-tea-cozy#2]
## **Use of HTML elements instead of classes.** [Original question](https://github.com/bootsfuehrerschein-kurse/prj-rev-bwfs-tea-cozy/blob/master/12_TeaCozy/ressources/css/styles.css#L34)

Yes, it's better to use the semantic HTML5 elements instead of a ``<div>`` with a class. Reason for this is because elements like ``<header>`` and ``<nav>`` are supposed to be _semantic_. This means that they carry some sort of meaning to them. The header is obviously the header and the nav is supposed to represent the navigation for your site. Using those elements in HTML5 is encouraged for numerous reasons (check [this article](http://html5doctor.com/lets-talk-about-semantics/) out, and also [this one](https://www.thoughtco.com/why-use-semantic-html-3468271) and if you still aren't convinced, check [this one](http://vanseodesign.com/web-design/semantic-html/) out too).

TL;DR
It is better to use them, so go ahead and use semantic HTML.
